packages/scandipwa/src/component/Breadcrumbs/Breadcrumbs.component.js

```diff
@@ -57,15 +59,24 @@ export class Breadcrumbs extends PureComponent {
         ));
     }
 
-    render() {
-        const { breadcrumbs, areBreadcrumbsVisible } = this.props;
+    shouldHideBreadcrumbs() {
+        const { areBreadcrumbsVisible } = this.props;
         const { pathname = appendWithStoreCode('/') } = location;
 
-        if (
+        const shouldHide = (
             !areBreadcrumbsVisible
-            || pathname.match(appendWithStoreCode('/account'))
+            || pathname.match(appendWithStoreCode(ACCOUNT_URL))
+            || pathname.match(appendWithStoreCode(CHECKOUT_URL))
             || isHomePageUrl(pathname)
-        ) {
+        );
+
+        return shouldHide;
+    }
```

Comment:
The `shouldHide` variable is unnecessary here.
You can simply return the value.

```javascript
return !areBreadcrumbsVisible
  || pathname.match(appendWithStoreCode(ACCOUNT_URL))
  || pathname.match(appendWithStoreCode(CHECKOUT_URL))
  || isHomePageUrl(pathname)
```